server/server.js

```diff
@@ -102,6 +102,8 @@ app.get('/api/getWatchlist', (req, res) => {
 })
 
 app.post('/api/postFriend', (req, res) => {
+  console.log('hi there');
+  console.log('req.body', req.body);
```

Comment:
This one too :)

Reply:
Thanks. Sometimes console logs are kinda like carrying on a conversation with yourself.